templates/templates/footer.html

```diff
@@ -1,9 +1,9 @@
 {% block footer %}
- 
+
 <footer class="p-footer u-no-margin--top">
   <div class="row">
     <div class="col-12">
-      <p>&copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical&nbsp;Ltd.</p>
+      <p>&copy; {% now "Y" %} Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical&nbsp;Ltd.</p>
```

Comment:
Can this be made as a seperate commit? A seperate PR is probably a better idea in general.

Reply:
Removed, I'll open another PR for this.